### PR TITLE
Login form

### PR DESCRIPTION
**Pull request by emmapaczkowski**
Allow users to save places to the Firebase real time database as well as add a field for users name in sign up form.

### CODE REVIEW

File: where-to-travel/src/main/webapp/script.js [hunk 1]
```diff
@@ -282,7 +282,7 @@ function populatePlaces(placeArray) {
     const directionsLink = 'https://www.google.com/maps/dir/' +
       home.lat + ',' + home.lng + '/' +
       coordinates.lat() + ',' + coordinates.lng();
-
+
```
Comment (saileshsimhadri) [inline where-to-travel/src/main/webapp/script.js]: remove whitespace

File: where-to-travel/src/main/webapp/script.js [hunk 2]
```diff
@@ -330,6 +330,23 @@ function populatePlaces(placeArray) {
   }
   $('.icon').click(function() {
     $(this).toggleClass('press');
+    if (firebase.auth().currentUser ) {
+      const name = $(this).parent().parent().parent().attr('placeName');
+      const link = $(this).parent().next().attr('href');
+      const time = $(this).parent().next().next().text();
+      console.log(name + " " + " " + link + " " + time);
```
Comment (saileshsimhadri) [inline where-to-travel/src/main/webapp/script.js]: Can the logs be removed?

File: where-to-travel/src/main/webapp/script.js [hunk 3]
```diff
@@ -553,7 +570,7 @@ function getPlacesFromDirection(lat, lng) {
         }
       }
       else {
-        console.log(status);
+        //console.log(status);
```
Comment (saileshsimhadri) [inline where-to-travel/src/main/webapp/script.js]: remove

**Review by saileshsimhadri — APPROVED** [review]

File: where-to-travel/src/main/webapp/script.js (hunk 1 above)
Comment (emmapaczkowski) [reply where-to-travel/src/main/webapp/script.js]: done

File: where-to-travel/src/main/webapp/script.js (hunk 3 above)
Comment (emmapaczkowski) [reply where-to-travel/src/main/webapp/script.js]: done

File: where-to-travel/src/main/webapp/script.js (hunk 2 above)
Comment (emmapaczkowski) [reply where-to-travel/src/main/webapp/script.js]: done

**Review by wjiamin — APPROVED** [review]

**Pull request merged**